core/Cargo.toml

```diff
@@ -21,7 +21,7 @@ std = ["math/std", "winter-utils/std"]
 
 [dependencies]
 math = { package = "winter-math", version = "0.6", default-features = false }
-crypto = { package = "miden-crypto", version = "0.6", default-features = false }
+crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto.git", branch = "frisitano-recording-map-finalizer", default-features = false }
```

Comment:
I doesn't seem like this change is required for this PR. If so, let's remove it.

Reply:
We need this for the `.finalize()` api changes for the `RecordingMap` here:

https://github.com/0xPolygonMiden/miden-vm/blob/0c15c62f3cdf872788b0a0b9b235206095abc2b5/processor/src/advice/providers.rs#L456-L457

Reply:
Ah - makes sense. Once https://github.com/0xPolygonMiden/crypto/pull/171 is merged, we can switch this to `next` and then merge this PR.